src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -522,9 +522,9 @@ default Path resolve(String other) {
      *
      * <p> If {@code more} does not specify any {@code Path}s,
      * then the result is {@link #resolve(Path) resolve(first)}. If
-     * {@code more} specifies one or more {@code Path}s, then each non-empty
-     * {@code Path}, including {@code first}, is resolved iteratively against
-     * this {@code Path}.
+     * {@code more} specifies one or more {@code Path}s, then {@code first}
+     * is resolved against this path and then the additional paths in
+     * {@code more} are iteratively resolved.
```

Comment:
Does the following better describe what this method does?
```suggestion
     * {@code more} are iteratively resolved against the previous path.
```